### PR TITLE
feat: add api-breaker plugin form

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
Add api-breaker plugin form.
image

**Related issues**

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/components/Plugin/UI/api-breaker.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+};
+
+const ApiBreaker: React.FC<Props> = ({ form }) => {
+  const { formatMessage } = useIntl()
+
+  return (
+    <Form
+      form={form}
+      {...FORM_ITEM_LAYOUT}
+      initialValues={{ unhealthy: { http_statuses: [500] }, healthy: { http_statuses: [200] } }}
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/api-breaker.tsx:49]: default value?

File: web/src/components/Plugin/UI/api-breaker.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    >
+      <Form.Item
+        label="break_response_code"
+        required
+        name="break_response_code"
+        tooltip={formatMessage({ id: 'component.pluginForm.api-breaker.break_response_code.tooltip' })}
+      >
+        <InputNumber min={200} max={599} required></InputNumber>
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/api-breaker.tsx:57]:
```suggestion
        <InputNumber min={200} max={599} required />
```

**Review by juzhiyuan — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Review by Yiyiyimu — APPROVED** [review]

File: web/src/components/Plugin/locales/zh-CN.ts [hunk 3]
```diff
@@ -29,6 +29,12 @@ export default {
   'component.pluginForm.api-breaker.unhealthy.failures.tooltip': '触发不健康状态的连续错误请求次数。',
   'component.pluginForm.api-breaker.healthy.http_statuses.tooltip': '健康时候的状态码。',
   'component.pluginForm.api-breaker.healthy.successes.tooltip': '触发健康状态的连续正常请求次数。',
+  // limit-conn
+  'component.pluginForm.limit-conn.conn.tooltip': '允许的最大并发请求数。超过 conn 的限制、但是低于 conn + burst 的请求，将被延迟处理。',
+  'component.pluginForm.limit-conn.burst.tooltip': '允许被延迟处理的并发请求数。',
+  'component.pluginForm.limit-conn.default_conn_delay.tooltip': '默认的典型连接（或请求）的处理延迟时间。',
```
Comment (juzhiyuan) [inline web/src/components/Plugin/locales/zh-CN.ts]: `典型`?

for those fields, need @membphis @starsz @nic-chen  to take a look

Comment (nic-chen) [reply web/src/components/Plugin/locales/zh-CN.ts]: I have never seen such a statement either

Comment (juzhiyuan) [reply web/src/components/Plugin/locales/zh-CN.ts]: 😂

File: web/src/components/Plugin/UI/api-breaker.tsx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      form={form}
+      {...FORM_ITEM_LAYOUT}
+      initialValues={{ unhealthy: { http_statuses: [500] }, healthy: { http_statuses: [200] } }}
+    >
+      <Form.Item
+        label="break_response_code"
+        name="break_response_code"
+        rules={[{
+          required: true,
+          message: `${formatMessage({ id: 'component.global.pleaseEnter' })} break_response_code`
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/api-breaker.tsx:56]: i18n

File: web/src/components/Plugin/UI/api-breaker.tsx [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+        tooltip={formatMessage({ id: 'component.pluginForm.api-breaker.break_response_code.tooltip' })}
+        validateTrigger={['onChange', 'onBlur', 'onClick']}
+      >
+        <InputNumber min={200} max={599} required />
+      </Form.Item>
+
+      <Form.Item
+        label="max_breaker_sec"
+        name="max_breaker_sec"
+        initialValue={300}
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/api-breaker.tsx:67]: Why use initialValue here?

File: web/src/components/Plugin/UI/api-breaker.tsx [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+        tooltip={formatMessage({ id: 'component.pluginForm.api-breaker.max_breaker_sec.tooltip' })}
+      >
+        <InputNumber min={60} />
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/api-breaker.tsx:70]: https://github.com/apache/apisix/blob/master/apisix/plugins/api-breaker.lua#L40

**Review by juzhiyuan — APPROVED** [review]
No urgent changes need to be updated, but those comments should resolve in the new PR.

File: web/src/components/Plugin/UI/api-breaker.tsx (hunk 6 above)
Comment (juzhiyuan) [reply web/src/components/Plugin/UI/api-breaker.tsx:70]: @LiteSun this one.

**Review by nic-chen — APPROVED** [review]

File: web/src/components/Plugin/UI/api-breaker.tsx (hunk 5 above)
Comment (LiteSun) [reply web/src/components/Plugin/UI/api-breaker.tsx:67]: use as default value.

**Pull request merged**